presto-raptorx/src/main/java/com/facebook/presto/raptorx/metadata/CommitCleaner.java

```diff
@@ -150,6 +152,19 @@ public synchronized void removeOldCommits()
             masterDao.cleanupTransactions(ids);
         }
 
+        // oldest worker successful transactions, for worker permenantly down
+        // TODO(taozhao) this may need to be a separate background job
```

Comment:
Nit: Remove your name in the TODO. We will see it in the blame :)

Reply:
Names removed